examples/example1

```diff
@@ -0,0 +1 @@
+../ui-tests/data/merge_test1
```

Comment:
What are the files in this examples folder meant to be used for? How does it differ from the files in the ui-tests folder, and the nbdime/tests/files folder?

Reply:
Maybe we could reduce the number of examples in the dedicated folders.They were used during the migration process to check various diff and merge configurations. 
The diff in `ui-tests/diff_test1` and  `ui-tests/diff_test2` are variants of example1 (center.ipynb with addition/deletions of additionnal cells) and `ui-tests/diff_test3` is a variant of `example8` (with `left.ipynb` and `center.ipynb` notebooks). 
The merge in `ui-tests/merge_test1` and `ui-tests/merge_test2` are variant of example1 and  `ui-tests/merge_test3` is a variant of `example3`. 

`nbdime/tests/files folder` has nothing in common with `ui-tests` and `examples`.